src/store/models/network.spec.ts

```diff
@@ -48,6 +48,15 @@ describe('Network model', () => {
   // helper to get the first network in the store
   const firstNetwork = () => store.getState().network.networks[0];
 
+  const createTaroNetwork = () => {
+    const network = getNetwork(1, 'test network', Status.Stopped, 2);
```

Comment:
I see you moved the test cases into the `Taro network` group. No you no longer need the `createTaroNetwork` function declared here. You can just move this code back into the `beforeEach`.